aries-site/src/examples/components/button/ButtonExample.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { Button } from 'grommet';
+import { Next } from 'grommet-icons';
+
+export const ButtonExample = () => {
+  return <Button label="Button" icon={<Next />} reverse onClick={() => {}} />;
```

Comment:
It doesn't feel "default" if we have to use `reverse` on it. I know this is adhering to designs but I'm wondering if it's not the best example for default.

Reply:
agree. I flip-flopped on it. Will take it out.